src/firebase/auth/useAuth.tsx

```diff
@@ -17,4 +17,4 @@ const useAuth = () => {
 	return { user: currentUser, role: currentUserRole }
 }
 
-export default useAuth
+export default useAuth
```

Comment:
before making a commit - always check the list of changed files to prevent changes like this in your PR.